spec/features/bidder_interacts_with_auction_spec.rb

```diff
@@ -85,4 +113,83 @@
     expect(page).to have_content("No bids yet.")
   end
 
+  scenario "Viewing bid history for running auction" do
+    Timecop.scale(36000) do
+      create_running_auction
+    end
+    path = auction_bids_path(@auction.id)
+    visit path
+
+    # sort the bids so that newest is first
```

Comment:
oh I see! So when I saw the comment "check the amount column" I thought it was referring to the DATABASE column. Thoughts on how that could be clarified either as a comment or code?